Make.inc

```diff
@@ -924,18 +926,20 @@ else
   UTF8PROC_INC := $(build_includedir)
 endif
 
-
 # BinaryBuilder options.  We default to "on" for all the projects listed in BB_PROJECTS,
 # but only if contrib/normalize_triplet.py works for our requested triplet
 ifeq ($(shell python $(JULIAHOME)/contrib/normalize_triplet.py $(or $(XC_HOST),$(XC_HOST),$(BUILD_MACHINE)) >/dev/null 2>/dev/null; echo $$?),0)
 USE_BINARYBUILDER := 1
 else
+ifneq ($(shell python $(JULIAHOME)/contrib/normalize_triplet.py x86_64-linux-gnu),x86_64-linux-gnu)
+$(warning python normalize_triplet.py appears to be non-functional, so BinaryBuilder disabled)
+endif
```

Comment:
Why isn’t the “echo $$?” sufficient?

Reply:
It is. This tests something entirely different.

Reply:
Can you give me a situation where the script runs successfully but does not recognize x86_64-Linux-gnu ?

Reply:
this is in the failure branch

Reply:
Ah, oh I see.  Got it, I was missing the `else` just above this.  Thanks.